nexus/db-model/src/utilization.rs

```diff
@@ -9,6 +9,7 @@ use uuid::Uuid;
 pub struct SiloUtilization {
     pub silo_id: Uuid,
     pub silo_name: Name,
+    pub silo_discoverable: bool,
```

Comment:
Strictly speaking, while this is fine to add, you don't actually need this line, right? In the query you're using it in SQL directly.

Reply:
Hmmm, perhaps. I thought this struct is what generated the type to use for the `dsl` but that might actually be from the `table!` call. Yeah, regardless, it shouldn't matter.